### PR TITLE
TRAN-56: Added sonarcloud workflow

### PR DESCRIPTION
**Pull request by dborowiecki**
TICKET: [TRAN-56](https://sd-transit.atlassian.net/browse/TRAN-56)

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[tox]
+envlist = py310
+skipsdist = True
+
+[testenv]
+deps =
+    -rrequirements.txt
```
Comment (sniedzielski) [inline tox.ini:7]: Is typo here? should be 'requirements.txt'?

**Review by sniedzielski — COMMENTED** [review]
Please check one comment, everything else looks good!

File: tox.ini (hunk 1 above)
Comment (dborowiecki) [reply tox.ini:7]: No, it's on purpose, see: [Tox doc for requirents.txt](https://github.com/SD-TransIT/transit_backend/pull/2#discussion_r938622215)

Comment (sniedzielski) [reply tox.ini:7]: ok

**Review by sniedzielski — APPROVED** [review]

**Pull request merged**